internal/sdkprovider/service/account/account_team_member.go

```diff
@@ -26,10 +26,14 @@ var aivenAccountTeamMemberSchema = map[string]*schema.Schema{
 		Description: userconfig.Desc("An account team id").ForceNew().Build(),
 	},
 	"user_email": {
-		Type:        schema.TypeString,
-		Required:    true,
-		ForceNew:    true,
-		Description: userconfig.Desc("Is a user email address that first will be invited, and after accepting an invitation, he or she becomes a member of a team.").ForceNew().Build(),
+		Type:     schema.TypeString,
```

Comment:
Perhaps we can use https://github.com/hashicorp/terraform-plugin-sdk/blob/main/helper/validation/strings.go#L93C6-L93C17 `validation.StringMatch()`?

Reply:
it's better to have it as a separate function to avoid duplicating this function call all around the code in multiple services